apps/kyb-app/src/components/organisms/UIRenderer/elements/JSONForm/components/TagsInput/TagsInput.tsx

```diff
@@ -0,0 +1,35 @@
+import { RJSFInputProps } from '@ballerine/ui';
+import { Tag, TagInput } from 'emblor';
+import { FunctionComponent, useMemo, useState } from 'react';
+
+export type ITagsInputProps = RJSFInputProps;
+
+export const TagsInput: FunctionComponent<ITagsInputProps> = ({
+  onBlur,
+  onChange,
+  formData,
+  id,
+}) => {
+  const [activeTagIndex, setActiveTagIndex] = useState<number | null>(null);
+
+  const tags = useMemo(() => {
+    if (!Array.isArray(formData)) return [];
+
+    return formData.map((tag, index) => {
+      return {
+        id: String(index),
+        text: tag,
+      };
```

Comment:
Likely TypeScript will understand its an array of tags.
```suggestion
      } satisfies Tag;
```